eth/gaspricemonitor.go

```diff
@@ -24,9 +24,12 @@ type GasPriceMonitor struct {
 	polling         bool
 	pollingInterval time.Duration
 	cancel          context.CancelFunc
+	// pollingMu is a mutex that protects access to polling related
+	// fields
```

Comment:
seems sort of trivial to have one word on a seperate line 😅

Reply:
Comment golf is fun: how concise can we make the comment while preserving clarity of the meaning? In this case, we can probably omit the first five words and say "Protects access to polling related fields"

Although the convention I like is to place this note preceding the fields that need to be protected; eg "The following fields need to be protected by the mutex `pollingMu`" and so forth; see an example here

https://github.com/livepeer/go-livepeer/blob/17701ca9a7bfb8b84de08da9fae1223ae2053b29/server/mediaserver.go#L90-L95

Reply:
Fixed in 2cec5ef